lib/compile/index.js

```diff
@@ -115,6 +123,7 @@ function compile(schema, root, localRefs, baseId) {
                    + sourceCode + 'return validate;';
 
     if (opts.beautify) {
+      loadBeautifyIfNeeded();
```

Comment:
Could we simplify a bit? Something like:

```javascript
if (beautify === undefined) {
  beautify = (function() {
    try { return require('' + 'js-beautify').js_beautify; }
    catch(e) { beautify = false }
  })();
}
```

No need to create two new names really...